src/Configuration/Loader/DistFileLoader.php

```diff
@@ -3,18 +3,21 @@
 
 namespace GrumPHP\Configuration\Loader;
 
+use Symfony\Component\Config\FileLocatorInterface;
+use Symfony\Component\Config\Loader\FileLoader;
 use Symfony\Component\Config\Loader\LoaderInterface;
 use Symfony\Component\Config\Loader\LoaderResolverInterface;
 
 /**
  * A decorating dist loader that supports **.dist files and defers loading.
  */
-final class DistFileLoader implements LoaderInterface
+final class DistFileLoader extends FileLoader implements LoaderInterface
```

Comment:
I'm a bit confused on why this solves your issue:

The idea of this class is to be a decorator around another loader.
Every public method inside this decorator proxies to the wrapped loader.

So how does extending the FileLoader helps solving the problem?

Reply:
In theory I'm not sure and you're right. In practice does this help, because the path to the .dist file is now resolved. Where the resolving didn't happen before.

I didn't take a deep dive into more code. I've added some var_dumps in all the load functions (from this DistFileLoader and the YamlFileLoader). Without the 'extend' the load function was presented with an unresolved path. With the 'extend' it was presented with a resolved path.

Reply:
In addition... if you think this is not the correct way, no probs. It resolved the issue for me. But yeah, there might be a better way.

Reply:
Probably one of the abstract FileLoader public methods is fixing the issue for you for some specific reason.

However I've got the feeling there should be a better fix indeed but haven't touched those classes in a long while so don't know without diving into it.

One thing I could think of, is that the import falls in this scenario which makes it fail:

https://github.com/phpro/grumphp/blob/16431b6c96a5e25a820f71dd17a0facefc4f5f86/src/Configuration/Loader/DistFileLoader.php#L35-L37